plugins/modules/guest_info.py

```diff
@@ -72,6 +71,12 @@
 
 from collections import defaultdict
 
+try:
+    from com.vmware.vcenter_client import VM
+except ImportError:
```

Comment:
I would write `get_vm_by_name` method in module utils, so we avoid this not-nice try-catch, it's handled there already